src/pipelines/rust_app.rs

```diff
@@ -238,6 +295,44 @@ impl RustApp {
             wasm_output: hashed_wasm_name,
         })
     }
+
+    async fn wasm_opt_build(&self, hashed_name: &str) -> Result<()> {
+        // Zero means no optimizations (valid arg for wasm-opt), so we can skip calling wasm-opt as
+        // it wouldn't have any effect.
+        if self.wasm_opt == WasmOptLevel::Off {
```

Comment:
```suggestion
        // If opt level is off, we skip calling wasm-opt as it wouldn't have any effect.
        if matches!(self.wasm_opt, WasmOptLevel::Off) {
```

Reply:
That's definitely a better description, especially now where I turned this into an enum 👍